packages/react/src/Grid/Grid.tsx

```diff
@@ -22,17 +22,77 @@ export type GridColumnNumbers = {
   wide: GridColumnNumber
 }
 
-interface GridComponent
-  extends ForwardRefExoticComponent<PropsWithChildren<HTMLAttributes<HTMLDivElement>> & RefAttributes<HTMLDivElement>> {
+type GridDensity = 'low' | 'high'
+
+type GridMarginSize = 'small' | 'medium' | 'large'
+
+type GridMarginVerticalProp = {
+  marginBottom?: never
+  marginTop?: never
+  /** The amount of vertical whitespace above and below the grid. */
```

Comment:
I'm not sure, but I think Intellisense only picks up JSDoc type comments in this format:

```suggestion
  /** 
  * The amount of vertical whitespace above and below the grid. 
  */
```

Reply:
Hm, JSDoc annotations don't show very reliably on my machine anyway... Let's keep it single line, if it causes issues we can always change it later